### ISSUE TITLE
breakage when branches look likes paths

### ISSUE THREAD

**Issue by dweomer** [dweomer/git-semver#5]
Need to handle more than simple branch names, i.e.:

```bash
git semver -debug init
# $GIT_DIR = $HOME/Projects/local-git/.git
# $GIT_WORK_TREE = $HOME/Projects/local-git
# $SEMVER_ORIGIN_NAME = origin
# $SEMVER_USER_EMAIL = semver@semver.org
# $SEMVER_USER_NAME = semver
# $SEMVER_BRANCH = herp/derp
# $SEMVER_TEMP = /var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-372124826
# git clone --branch semver $HOME/Projects/local-git/.git $SEMVER_TEMP
# -> Clone(): remote repository is empty
# git init /var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-372124826
git-semver: open /var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-372124826/herp/derp: no such file or directory
```

**Comment by dweomer** [comment]
```bash
$ git init
Initialized empty Git repository in /tmp/derp/.git/
$ git checkout -b what/the/💩
Switched to a new branch 'what/the/💩'
$ git semver -debug init
# $GIT_DIR = /tmp/derp/.git
# $GIT_WORK_TREE = /tmp/derp
# $SEMVER_REMOTE_NAME = origin
# $SEMVER_USER_EMAIL = semver@semver.org
# $SEMVER_USER_NAME = semver
# $SEMVER_BRANCH = what/the/💩
# $SEMVER_TEMP = /var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-423578668
# git clone --branch semver /tmp/derp/.git $SEMVER_TEMP
# -> Clone(): remote repository is empty
# git init /var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-423578668
# git push /tmp/derp/.git semver
# '/var/folders/7s/bwc8hz3n5498r68fw4gq6cxr0000gq/T/semver-423578668' -> '/tmp/derp/.semver'
# $SEMVER_DIR = /tmp/derp/.semver
$ git log semver
commit 9bce7d23bdb14f725a3b26ba35ba79db495fe40b (semver)
Author: semver <semver@semver.org>
Date:   Mon Mar 25 11:05:37 2019 -0700

    semver(what/the/💩): 0.0.0
```

**Comment by dweomer** [comment]
fixed by #7
